packages/swapkit/helpers/src/helpers/asset.ts

```diff
@@ -80,6 +80,7 @@ export const gasFeeMultiplier: Record<FeeOption, number> = {
 export const isGasAsset = ({ chain, symbol }: { chain: Chain; symbol: string }) => {
   switch (chain) {
     case Chain.Arbitrum:
+      return symbol === "ETH";
```

Comment:
you dont need an extra return here

Reply:
Yeah - it should be handled directly by Chain.Optimism.
```suggestion
```